packages/terra-progress-bar/src/ProgressBar.scss

```diff
@@ -1,4 +1,17 @@
 :local {
+
+  @mixin terra-gradient($from-color, $to-color) {
+    background:
+      linear-gradient(to left, transparent, rgba(255, 255, 255, 0.5)),
+      linear-gradient(to left, $from-color, $to-color);
+  }
+
+  .progress-bar-gradient {
```

Comment:
Add `&::-ms-fill` within this selector:

e.g.
```
.progress-bar-gradient {
    &::-webkit-progress-value {
      @include terra-gradient(currentColor, currentColor);
    }

    &::-ms-fill {
      @include terra-gradient(currentColor, currentColor);
    }
  }
```

This will enable gradient support in IE 10, 11, and Edge.

<img width="1369" alt="screen shot 2018-03-28 at 11 17 05 am" src="https://user-images.githubusercontent.com/633148/38042459-0af0d85e-327a-11e8-81f6-1f0df5a7ac2c.png">
<img width="1366" alt="screen shot 2018-03-28 at 11 16 04 am" src="https://user-images.githubusercontent.com/633148/38042460-0aff8700-327a-11e8-8eec-06b18737f443.png">
<img width="1276" alt="screen shot 2018-03-28 at 11 17 24 am" src="https://user-images.githubusercontent.com/633148/38042458-0ae02ffe-327a-11e8-8f2d-4092f1b02d27.png">

Reply:
Thanks @bjankord Added `&::-ms-fill` selector